src/app/handlers/command.handler.ts

```diff
@@ -2,19 +2,22 @@ import { IMessage, IContainer, IHandler } from '../../common/types';
 import Constants from '../../common/constants';
 
 export class CommandHandler implements IHandler {
-  constructor(public container: IContainer) {}
+  constructor(public container: IContainer) { }
 
   public async execute(message: IMessage): Promise<void> {
-    const command = this.build(message.content);
+    const alias = this.build(message.content);
     const plugins = this.container.pluginService.plugins;
-    if (!command) {
+    const aliases = this.container.pluginService.aliases;
+
+    if (!alias) {
       return;
     }
```

Comment:
Log an error

Reply:
Why would this be an error, all that means is the message is not to be read as a command?

Reply:
changed it to `(command === undefined)`, and left a comment to explain it, but as I said I don't see a reason to log it.  Since every message that doesn't start with ! will trigger a warning if we did that.